common/web/gesture-recognizer/tsconfig.json

```diff
@@ -11,9 +11,6 @@
     //       but existing code in web/ breaks some of these settings
     //
     // At present, the code actually compiles without them... but the TS tests are a different matter.
-    "noImplicitAny": false,
-    "noUnusedLocals": false,
```

Comment:
The `"noUnusedLocals"` bit should've been removed earlier, but must have gotten mangled a bit in all the rebasing to reorder... everything else.